### PR TITLE
FEM: lots of small fixes, typos, unit test improvements, package improvements

### PR DESCRIPTION
**Pull request by berndhahnebach**
- [x] Branch rebased on latest master `git pull --rebase upstream master`
- [x] Unit tests confirmed to pass by running `./bin/FreeCAD --run-test 0`
- [x] Commit message is [well-written](https://chris.beams.io/posts/git-commit/)

- no new features
- only changes in FEM
- see PR title

bernd

### CODE REVIEW

**Comment by berndhahnebach** [comment]
Travis does not run. I had the same problem at home. For some reason I had to delete all build files and modules in Fem build directory. Afterwards unit  tests worked well. Is it possible to do this manually in travis. We do run cached builds, which mean we may have the same problem ? May be wait and see how win builds do ...

bernd

**Comment by berndhahnebach** [comment]
Appveyor fails too. I will have a look ...

File: src/Mod/Fem/femsolver/calculix/tasks.py [hunk 1]
```diff
@@ -69,6 +69,10 @@ def run(self):
             c.beam_sections, c.shell_thicknesses, c.fluid_sections,
             self.solver.AnalysisType, self.directory)
         path = w.write_calculix_input_file()
+        if path is not None:
```
Comment (luzpaz) [inline src/Mod/Fem/femsolver/calculix/tasks.py]: perhaps a comment should precede this section of the code like:  
`# report ot user if task succeeded`?

Comment (berndhahnebach) [reply src/Mod/Fem/femsolver/calculix/tasks.py]: https://github.com/FreeCAD/FreeCAD/pull/1210/commits/c6740dc89fe802861a25947fa8633ae55ee268f9

**Pull request merged**